src/main/java/edu/harvard/iq/dataverse/authorization/providers/builtin/BuiltinUser.java

```diff
@@ -43,7 +44,7 @@
 
     @NotBlank(message = "Please enter a username.")
     @Size(min=2, max=60, message ="Username must be between 2 and 60 characters.")
-    @Pattern(regexp = "[a-zA-Z0-9\\_\\-\\.]*", message = "Found an illegal character(s). Valid characters are a-Z, 0-9, '_', '-', and '.'.")
+    @ValidateUserName(message = "Found an illegal character(s). Valid characters are a-Z, 0-9, '_', '-', and '.'.")
```

Comment:
If you look at the validateUserName method in DataverseUserPage line 220 shows that the error message displayed on the page comes from the Bundle.